build_tools/cmake/iree_cross_compile.cmake

```diff
@@ -118,6 +118,13 @@ function(iree_create_configuration CONFIG_NAME)
         -DIREE_BUILD_PYTHON_BINDINGS=${_CONFIG_BUILD_PYTHON_BINDINGS}
         -DIREE_BUILD_JAVA_BINDINGS=${_CONFIG_BUILD_JAVA_BINDINGS}
         -DIREE_BUILD_EXPERIMENTAL=${_CONFIG_BUILD_EXPERIMENTAL}
+        -DIREE_ENABLE_ASAN=${IREE_ENABLE_ASAN}
+        -DIREE_ENABLE_MSAN=${IREE_ENABLE_MSAN}
+        -DIREE_ENABLE_TSAN=${IREE_ENABLE_TSAN}
+        -DIREE_ENABLE_CCACHE=${IREE_ENABLE_CCACHE}
+        -DLLVM_CCACHE_BUILD=${LLVM_CCACHE_BUILD}
+        -DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
+        
```

Comment:
Can you move these above the `# LINT.IfChange(iree_cross_compile_invoke)`? We don't have ifchange/thenchange running on GitHub, but the general idea is that the block of code here should map 1:1 to the code in the other files:

https://github.com/google/iree/blob/746eeece68220a4de7f2d351911198821b08f8b9/CMakeLists.txt#L36-L57

https://github.com/google/iree/blob/746eeece68220a4de7f2d351911198821b08f8b9/build_tools/cmake/iree_cross_compile.cmake#L77-L98

Reply:
thanks! done.